packages/validation-helpers/src/schemas/ui_schemas.ts

```diff
@@ -10,3 +10,13 @@ export const loginFormSchema = z.object({
 });
 
 export type LoginFormType = z.infer<typeof loginFormSchema>;
+
+export const userDataFormSchema = z.object({
```

Comment:
Acc. to discussions with Mikko, these schemas should probably be moved as close to the Form components as possible, ie. in the file where the Form is specified (UserDataForm.tsx, LoginForm.tsx), for easier readability and for not to obfuscate the code unnecessarily.